runtime/_data.ts

```diff
@@ -5,27 +5,12 @@ export const sidebar = [
     title: "Getting Started",
     items: [
       {
-        label: "Quick Start",
-        id: "/runtime/manual/",
-      },
-      {
-        label: "Deno Basics",
-        items: [
-          "/runtime/manual/getting_started/first_steps/",
-          "/runtime/manual/getting_started/setup_your_environment/",
-          "/runtime/manual/getting_started/command_line_interface/",
-          "/runtime/manual/getting_started/configuration_file/",
-          "/runtime/manual/getting_started/web_frameworks/",
```

Comment:
is the web_frameworks document orphaned after this change?

Reply:
it will be moved into "Fundamentals" section and Tutorials section in future updates to this branch